gui/main.js

```diff
@@ -0,0 +1,60 @@
+
+let { app } = require('electron')
+
+const HomeBrowserWindow = require('./src/home/home-browser-window')
+const ViewerBrowserWindow = require('./src/viewer/viewer-browser-window')
+const EventBus = require('./src/util/event-bus')
+
+let homeWindow = undefined
+const appConfig = {
+    title: "APK Dependency Graph",
+    devMode: true
```

Comment:
what is `devMode` and should it really be enabled?

Reply:
devMode property is to enable/disable the following menu and the developer options (for example.: Developer tools, reload...):
![screenshot from 2019-01-16 11-50-43](https://user-images.githubusercontent.com/1884044/51256940-5772d900-1985-11e9-97c3-5d74e233171d.png)

I set it as true because this helps me to develop. I am going to change it to false.